### PR TITLE
Release 2.3.2

### PR DESCRIPTION
**Pull request by d4straub**
This is release 2.3.2, which is mostly about bug fixes. For details see [CHANGELOG](https://github.com/nf-core/ampliseq/blob/dev/CHANGELOG.md).

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
> nf-core branch protection / test (pull_request_target) 

was a bug in the previous nf-core tools release and I hope will be resolved when this PR is merged.

**Review by erikrikarddaniel — APPROVED** [review]
Looks fine.

File: conf/modules.config
Comment (marissaDubbelaar) [inline conf/modules.config:75]: Don't you want to give the parameter `-a` and `-A` a separate line?

**Review by marissaDubbelaar — APPROVED** [review]
LGTM

File: conf/modules.config
Comment (d4straub) [reply conf/modules.config:75]: Possible, but those two are required to come in a pair, and thats not a bad thing when this is also visually clear here I think.

**Comment by d4straub** [comment]
Thanks for the review! Only waiting for the AWS megatest to be successful before I merge!

**Pull request merged**
